src/WebJobs.Script/Description/Workers/Rpc/MultiLanguageFunctionDescriptorProvider.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Azure.WebJobs.Script.Extensibility;
+using Microsoft.Azure.WebJobs.Script.Workers;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    internal class MultiLanguageFunctionDescriptorProvider : WorkerFunctionDescriptorProvider
```

Comment:
XML comments on all classes and methods?

Reply:
Adding XML Doc to all the new classes that i create. To keep the XML Doc consistent in a class.